src/dbcp/commands/publish.py

```diff
@@ -219,6 +256,12 @@ def to_yaml(self) -> str:
     is_flag=True,
     help="Upload the outputs to BigQuery",
 )
+@click.option(
+    "--upload-to-postgres",
+    default=False,
+    is_flag=True,
+    help="Upload the FYI outputs to Postgres",
```

Comment:
@zschira this PR seems to upload all data mart tables to Postgres, or is still just FYI? If not, we should change this comment.